src/screens/CreatePost/elements/UserProfile.js

```diff
@@ -19,15 +19,20 @@ const styles = StyleSheet.create({
   },
   profile: {flexDirection: 'row', alignItems: 'center'},
   username: {
-    color: COLORS.black,
+    color: colors.black,
     fontFamily: fonts.inter[600],
     fontWeight: 'bold',
     fontSize: 14
   },
   desc: {
     fontFamily: fonts.inter[400],
     fontSize: 12,
-    color: COLORS.gray8
+    color: colors.gray
+  },
+  switch: {
+    fontFamily: fonts.inter[400],
+    fontSize: 12,
+    color: colors.gray
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color properties in the styles for `username`, `desc`, and `switch` have been updated from `COLORS.*` to `colors.*`. This change is consistent with the PR description and there are no issues with these changes.
```diff
-    color: COLORS.black,
...
-    color: COLORS.gray8
...
-    color: COLORS.gray8
+    color: colors.black,
...
+    color: colors.gray
...
+    color: colors.gray
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->